### ISSUE TITLE
I've created completed solution files for this course

### ISSUE THREAD

**Issue by andrewbeldean** [MicrosoftLearning/20532-DevelopingMicrosoftAzureSolutions#159]
Hi all..

I've created a completed solution file for each module to allow students who perhaps do not have the required knowledge to open side by side with the lab. All are tested and working.

They are in each Mod folder labelled solution

Please feedback any issues.

Available for download from

https://s3-us-west-2.amazonaws.com/azure20532solutionfiles/Solution+Files.zip

**Comment by denschork** [comment]
Any one know if this course is planned for update?
Would be nice to get away from Web Roles. Maybe fully move into App Service Model using Web Jobs and Functions? Add 2 more modules and make it a full 5 day course.

**Comment by ConcreteGannet** [comment]
Thanks @andrewbeldean .

Could you create a pull request with your changes?

Cheers

Paul Foxworthy

**Comment by seesharprun** [comment]
The lab has already been updated to use Web Apps instead of Web Roles. I can create solution files but it's impossible to make solution files that will work "out of the box" without tweaking a few things. This is because:

- The database needs to be populated
- The connection strings and app settings need to point to live services.

I will mark this as something to consider in the next milestone.

**Comment by ConcreteGannet** [comment]
Connection strings and app settings aren't a big deal. The 20486 course has starters and solutions that won't work until you fix up connection strings for your own Azure services.

If the database isn't populated, could there be an error message to say specifically what's wrong?

**Comment by seesharprun** [comment]
Closing this was an accident. I added the **next-update** milestone tag to indicate that this should be done as part of the next major revision.

If we do it while bug fixes are in-flight, the solutions are likely to change before the solutions are done. For example, the post at the top of this thread has solutions that are no longer accurate to the starter projects.

This is definitely part of the punch list for the next major revision of this course. Sorry for causing confusion.
